src/frontend/sharedComponents/DeviceCard.tsx

```diff
@@ -30,16 +24,16 @@ export const DeviceCard = ({
   deviceId,
   dateAdded,
   onPress,
+  isConnected,
```

Comment:
I assume the default here is `true`. Could we set that?

```suggestion
  isConnected = true,
```